include/imu_ros2/iio_wrapper.h

```diff
@@ -124,6 +124,37 @@ class IIOWrapper {
     void unload();
     bool isBufferLoaded();
 
+    // members from adis16577-3
```

Comment:
this comment is really specific and can be removed
we want iio wrapper to be generic and to support other devices also, anything from adis1657x family